.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: scala
+sudo: false
+jdk: oraclejdk8
+
+scala:
+  - 2.10.6
+
+script:
+  - sbt "++$TRAVIS_SCALA_VERSION" publishLocal
+  - sbt "++$TRAVIS_SCALA_VERSION" test
+  - sbt "++$TRAVIS_SCALA_VERSION" scripted
+
```

Comment:
There's an [sbt plugin](https://github.com/sbt/sbt-doge) that does precisely this for sbt 0.13.x. In sbt 1.0.x, it is built in. I can either enable this plugin right now, or wait until switching to 1.0.x.